test/Nancy.Swagger.Tests/Services/DefaultSwaggerModelCatalogTest.cs

```diff
@@ -0,0 +1,79 @@
+﻿using FakeItEasy;
+using Nancy.Swagger.Services;
+using Swagger.ObjectModel;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Nancy.Swagger.Tests.Services
+{
+    public class DefaultSwaggerModelCatalogTest
+    {
+        private DefaultSwaggerModelCatalog _defaultSwaggerModelCatalog;
+
+        public DefaultSwaggerModelCatalogTest()
+        {
+            var fakeDataProviders = A.Fake<IEnumerable<ISwaggerModelDataProvider>>();
+
+            this._defaultSwaggerModelCatalog = new DefaultSwaggerModelCatalog(fakeDataProviders);
+        }  
+
+        [Theory]
+        [InlineData(typeof(int))]        
+        [InlineData(typeof(float))]
+        [InlineData(typeof(string))]
+        [InlineData(typeof(SwaggerFile))]
+        public void GetModelForType_Should_Return_Null_When_Type_Is_Primitive(Type type)
+        {
+            Assert.Null(_defaultSwaggerModelCatalog.GetModelForType(type));
+        }
+
+        [Fact]        
+        public void GetModelForType_Should_Return_SwaggerModelData_When_Type_Is_Not_Primitive()
+        {
+            Assert.NotNull(_defaultSwaggerModelCatalog.GetModelForType(typeof(TestModel)));
```

Comment:
Can we assign the result of GetModelForType to a variable and assert against that? Then the tests will have a consistent pattern

Reply:
OK ! I will modify it later !